tests/expressions/models.py

```diff
@@ -6,10 +6,15 @@
 from django.db import models
 
 
+class Manager(models.Model):
+    name = models.CharField(max_length=50)
+
+
 class Employee(models.Model):
     firstname = models.CharField(max_length=50)
     lastname = models.CharField(max_length=50)
     salary = models.IntegerField(blank=True, null=True)
+    manager = models.ForeignKey(Manager, models.CASCADE, null=True)
```

Comment:
Looks like adding this field changed the way some database returned unordered sets. You might need to add a `Meta.ordering = 'pk'` here.

Reply:
Thanks again @charettes.
As you mentioned, I tried to fix the problem by adding `Meta.ordering = 'pk'`, but that will fail some other tests which use `group by`. like:
https://github.com/django/django/blob/6fbce45b0376f0ec8f3bf244f4f1f8d62c201f58/tests/expressions/test_queryset_values.py#L43

So, I've decided to handle it in the new test and add the manager object inside the test.

Reply:
Makes sense, thanks Hasan!